Alignment/CommonAlignmentMonitor/plugins/AlignmentMonitorMuonSystemMap1D.cc

```diff
@@ -188,11 +188,9 @@ void AlignmentMonitorMuonSystemMap1D::book()
 {
   std::string wheel_label[5]={"A","B","C","D","E"};
 
-  for (int station = 1; station<=4; station++)
+  for (unsigned char station = 1; station<=4; station++)
```

Comment:
Well, the `station` seems to be never below 0 thus let's be more explicit about it. It also should allow to catch overflow with static and run-time analyzers. It's also counting just to 4, thus a single `char` can handle that. Also `char` is `signed` or `unsigned` based on architecture you are compiling thus specifying it explicitly provides more portable code. Also in string formats, like `sprintf(c_station, "%d", station);`, the compiler would complain that larger buffer is required, because `int` can hold higher decimal value. At the end, I think, I switched to use `std::string` instead of `char[]` so this could be also `unsigned int`. Also because `char` only cost 8 bytes you don't need to use full 32-bit register at least on `x86_64`/`amd64`, while it has not benefits on `aarch64` where minimal register access width is 32-bit. The more you  can fit into registers the less spilling and filling you will have, i.e. saving value to memory and loading it later back to do the operation.

I can modify to whichever is preferred.

Reply:
ok, understood. there are two changes:
signed -> unsigned (to be explicit)
int -> char (to reduce the buffer size requirement)
I am fine with that change, then.

Reply:
Correct and thanks!